main.css

```diff
@@ -63,6 +63,11 @@ body {
     width: 100%;
 }
 
+.vid-iframe {
+    height: 100%;
```

Comment:
Maybe height should not be 100% before it was not an issue because the attributes in `iframe` were not actually set it to 100%. But now it is, perhaps you can change this before merge to master. 
The only thing though, it is only in firefox it sets the height to 100% fully, however, I believe you'd be able to figure it out.

![Firefox iframe](https://user-images.githubusercontent.com/31193823/57179370-445c1a00-6e4b-11e9-993d-bc2088e6760f.png)


I'd still give the project a 👍